front/app/containers/App/index.tsx

```diff
@@ -235,7 +237,14 @@ const App = ({ children }: Props) => {
         scope.setUser({
           id: authUser.data.id,
         });
+        setAppContainerClassName(
```

Comment:
Hmm, why do we do this inside of this `configureScope` function related to sentry?